robot/EDA/resources/EDA.py

```diff
@@ -274,7 +297,38 @@ def java_click_on_element(self,path, *args, **kwargs):
             self.selenium.driver.find_element_by_xpath(main_loc))
         time.sleep(1)
 
+    def select_frame_with_value(self, value):
+        """ Selects frame identified by the given value
+            value should be the 'id', 'title' or 'name' attribute value of the webelement used to identify the frame
+        """
+        locator = eda_lex_locators["frame"]
+        locator = self.format_all(locator, value)
+        self.selenium.select_frame(locator)
+
+    def format_all(self, loc, value):
```

Comment:
Is this a locator builder?

Reply:
This is a way to select the needed frame, so that the locator can be resolved.